### DISCUSSION TITLE
Two same documents retrieved by bm25 for the same question have different score！

### DISCUSSION THREAD

**Discussion by cingtiye** [castorini/pyserini#1340] [Q&A] [answered]
Hi,

When I used pyserini to retrieve my question, I was surprised to find that two same documents retrieved by bm25 for the same question have different score.

![image](https://user-images.githubusercontent.com/32251546/201298392-9bb560e3-96a8-4f0f-b5c8-58de27d53268.png)
     
Could you please tell me how to avoid this problem?
      
Thank you!

**Comment by lintool** [comment, marked as answer]
Hi @cingtiye - have you checked your corpus? there are likely two underlying documents with the same id...

**Reply by cingtiye** [reply]
I check my corpus and I find there have two same id for defferent documents.
Thank you.
